Makefile

```diff
@@ -454,8 +454,9 @@ operator-sdk:
 bundle: kustomize operator-sdk manifests set-image-controller api-docs
 	$(OPERATOR_SDK) generate kustomize manifests -q
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
-	$(OPERATOR_SDK) bundle validate ./bundle
 	./hack/ignore-createdAt-bundle.sh
+	./hack/add-openshift-annotations.sh
+	$(OPERATOR_SDK) bundle validate ./bundle
```

Comment:
note that bundle generation is moved after `./hack/ignore-createdAt-bundle.sh`. IIRC the ignore should be executed after the bundle generation

Reply:
Maybe I'm not understanding something:
1. The bundle is generated:
```Makefile
	$(OPERATOR_SDK) generate kustomize manifests -q
	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
```
2. Ignore the `createAt` field if it was the only change:
```Makefile
	./hack/ignore-createdAt-bundle.sh
```
3. Add the OpenShift annotations:
```Makefile
	./hack/add-openshift-annotations.sh
```
4. Validate the generated bundle:
```Makefile
	$(OPERATOR_SDK) bundle validate ./bundle
```

> IIRC the ignore should be executed after the bundle generation
It is executed after the bundle generation.

Reply:
oh got it, the validation is executed afterwards not the generation :)